lib/cdo/github.rb

```diff
@@ -39,6 +39,26 @@ def self.merge_pull_request(pr_number)
     response['merged']
   end
 
+  # Creates and merges a pull request.
+  # @param base [String] The base branch of the requested pull request.
+  # @param head [String] The head branch of the requested pull request.
+  # @param title [String] The title of the requested pull request.
+  # @return [nil | Integer] The PR number of the newly created DTT if successful
+  #   or nil if unsuccessful.
+  def self.create_and_merge_pull_request(base:, head:, title:)
+    pr_number = create_pull_request(base: base, head: head, title: title)
+    success = merge_pull_request(pr_number)
```

Comment:
Do we need to manually check whether `pr_number` came back `nil` here, or will `merge_pull_request` just raise if passed nil?

Related: It'd be nice to use [`@raise`](http://www.rubydoc.info/gems/yard/file/docs/Tags.md#raise) to document possible failure states for these wrapper methods.